usagov/usagov-2021
# Usagov 1845 federal directory crypess test

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-1845
Update test to run in Englsih and Spanish from in one file
## Description


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps
Goto automated_tests/e2e-cypress/cypress/e2e/regression_testing/federal_directory.cy.js and run test to make sure is passes in English and Spanish

<img width="1163" alt="Screenshot 2024-09-17 at 1 01 39 PM" src="https://github.com/user-attachments/assets/17f3989e-07f8-4473-b984-8e0b9302fbda">

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
